lib_0.7/paraswap/V4/AugustusSwapper.md

```diff
@@ -37,11 +43,14 @@ In single path swaps no intermediary tokens are involved. Suppose user wants to
 Any important point to keep in mind is that only whitelisted exchanges can be used in a swap.
 
 ## Usage of Gas Tokens
+
 Gas tokens are used in each transaction to reduce the overall gas consumpation for the user. This will result in the reduced gas usage for the user when they paraswap.io.
+
 The gas tokens are held in the wallet owned by paraswap.io and consumed during the transaction by TokenTransferProxy contract. AugustusSwapper contract will calculate how much of gas tokens should be freed in order to optimize the gas consumption for any transaction and will then pass on that value to the TokenTransferProxy contract in order to free the gas tokens.
 
 ## Pausable
-The AugustusSwapper contract must be pausable in nature. This feature is required to ensure that the paraswap.io team is able to pause contract in case any severe attack vectors are found in the contract. This will allow them to render contract unusable for future trades and deploy the fixed version of the contract
+
+The AugustusSwapper contract must be pausable in nature. This feature is required to ensure that the paraswap.io team is able to pause contract in case any severe attack vectors are found in the contract. This will allow them to render contract unusable for future trades and deploy the fixed version of the contract.
```

Comment:
```suggestion
The AugustusSwapper contract must be pausable in nature. This feature is required to ensure that the paraswap.io team is able to pause contract in case any severe attack vectors are found in the contract. This will allow them to render contract unusable for future trades and deploy the fixed version of the contract
```